src/get_services_by_team.py

```
@@ -0,0 +1,24 @@
+import requests
+
+
+
+def _fetch_services_by_team(team_id: str, api_url: str):
+    """
+    Calls the service atlas api to get services for a team
+    :param team_id:
+    :param api_url:
+    :return:
+    """
+    response = requests.get(f"{api_url}/teams/{team_id}/services")
```

Ruff (0.14.6)

Probable use of `requests` call without timeout

(S113)